src/module/canvas/token/object.ts

```diff
@@ -38,16 +38,15 @@ class TokenPF2e<TDocument extends TokenDocumentPF2e = TokenDocumentPF2e> extends
         return canvas.visibility.testVisibility(this.center, { tolerance, object: this });
     }
 
+    get _animation(): Promise<boolean> | void {
```

Comment:
I take it they removed this getter, but make it less javascripty if you want to keep it around (no underscore, etc.).

Reply:
Done